websites/B/Bots For Discord/presence.ts

```diff
@@ -0,0 +1,64 @@
+const presence = new Presence({
+    clientId: "806926545771167774"
+});
+
+presence.on("UpdateData", async () => {
+    const presenceData: PresenceData = {
+        largeImageKey: "logo"
+    },
+        browsingStamp = Math.floor(Date.now() / 1000);
```

Comment:
you should probs init it outside of the updatedata event.

Reply:
Fixed 👌